### PR TITLE
Pytorch MAML fix for more than two workers with discrete actions

### PR DESCRIPTION
**Pull request by ChaceAshcraft**
## Why are these changes needed?

I couldn't run MAML with Pytorch was more than two workers. Seemed to be a default setting was conflicting with the implementation. 

I updated indexing in the meta-update call from 'self.inner_adaptation_steps' to 'self.inner_adaptation_steps-1' because it indexes into 'obs' which is 0-indexed, but self.inner_adapation_steps starts at 1. Also updated the default value for 'split' in the maml_loss __init__ to be based on the inner adaptation steps, the number of workers, and the batch size. The default value seemed to conflict with indexing in the loop in the MAMLLoss object. 

I'll work on the checks today and update as they are completed. 
- Pushed changes from `scripts/format.sh`
- Ran pytest on rllib tests folder and it did not pass, but seemed unrelated to these changes?

## Related issue number

I did not create and issue for this. 

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
   - No doc changes anticipated.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
   - See above about testing
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/agents/maml/maml_torch_policy.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import logging
 
 import ray
-from ray.rllib.evaluation.postprocessing import compute_gae_for_sample_batch, \
-    Postprocessing
+from ray.rllib.evaluation.postprocessing import Postprocessing
```
Comment (sven1977) [inline rllib/agents/maml/maml_torch_policy.py:4]:
```suggestion
from ray.rllib.evaluation.postprocessing import compute_gae_for_sample_batch, \
    Postprocessing
```

File: rllib/agents/maml/maml_torch_policy.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 from ray.rllib.policy.policy_template import build_policy_class
 from ray.rllib.policy.sample_batch import SampleBatch
-from ray.rllib.agents.ppo.ppo_tf_policy import setup_config
+from ray.rllib.agents.ppo.ppo_tf_policy import postprocess_ppo_gae, \
+    setup_config
```
Comment (sven1977) [inline rllib/agents/maml/maml_torch_policy.py:8]:
```suggestion
from ray.rllib.agents.ppo.ppo_tf_policy import setup_config
```

File: rllib/agents/maml/maml_torch_policy.py [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 from ray.rllib.agents.ppo.ppo_torch_policy import vf_preds_fetches, \
     ValueNetworkMixin
+from ray.rllib.agents.a3c.a3c_torch_policy import apply_grad_clipping
```
Comment (sven1977) [inline rllib/agents/maml/maml_torch_policy.py:11]:
```suggestion
```

Comment (sven1977) [inline rllib/agents/maml/maml_torch_policy.py:11]:
```suggestion
from ray.rllib.utils.torch_ops import apply_grad_clipping
```

File: rllib/agents/maml/maml_torch_policy.py [hunk 4]
```diff
@@ -355,7 +361,7 @@ def setup_mixins(policy, obs_space, action_space, config):
     stats_fn=maml_stats,
     optimizer_fn=maml_optimizer_fn,
     extra_action_out_fn=vf_preds_fetches,
-    postprocess_fn=compute_gae_for_sample_batch,
+    postprocess_fn=postprocess_ppo_gae,
```
Comment (sven1977) [inline rllib/agents/maml/maml_torch_policy.py:364]:
```suggestion
    postprocess_fn=compute_gae_for_sample_batch,
```

**Comment by ChaceAshcraft** [comment]
@sven1977 Do you need me to do anything more in order to finish up the PR and merge?

**Review by sven1977 — APPROVED** [review]
Thanks for the fix @ChaceAshcraft !

**Comment by sven1977** [comment]
@ChaceAshcraft , done! :)

**Comment by ChaceAshcraft** [comment]
Thanks again! Glad to help!

**Pull request merged**